orttraining/orttraining/python/training/ortmodule/torch_cpp_extensions/aten_op_executor/setup.py

```diff
@@ -199,48 +124,39 @@ class ATenOperatorCache {
         has_transform_func = true;
       }
     }
-
     if (!has_transform_func) {
       arguments[index] =
           aten_op.is_optional_arguments[index] ? c10::IValue(c10::optional<at::Tensor>(tensor)) : c10::IValue(tensor);
     }
   }
-
   SetIValueArguments<int64_t>(int_arguments, aten_op.is_optional_arguments, arguments);
   SetIValueArguments<float>(float_arguments, aten_op.is_optional_arguments, arguments);
   SetIValueArguments<bool>(bool_arguments, aten_op.is_optional_arguments, arguments);
   SetArrayIValueArguments<int64_t>(int_array_arguments, aten_op.is_optional_arguments, arguments);
   SetArrayIValueArguments<float>(float_array_arguments, aten_op.is_optional_arguments, arguments);
   SetArrayIValueArguments<bool>(bool_array_arguments, aten_op.is_optional_arguments, arguments);
-
   torch::jit::Stack stack;
   for (size_t i = 0; i < arguments.size(); i++) {
     torch::jit::push(stack, arguments[i]);
   }
-
   aten_op.op->getOperation()(&stack);
   std::vector<DLManagedTensor*> result;
   for (const auto& ret : torch::jit::pop(stack, aten_op.return_size)) {
     const auto& tensor = ret.toTensor();
     result.emplace_back(at::toDLPack(tensor.is_contiguous() ? tensor : tensor.contiguous()));
   }
-
   return result;
 }
-
 size_t execute_aten_operator_address() { return reinterpret_cast<size_t>(&ExecuteATenOperator); }
-    """
 
-    aten_op_executor_cpp_extension = load_inline(name='aten_op_executor', cpp_sources=[aten_op_executor_cpp_source],
-                                                 extra_cflags=['-D__HIP_PLATFORM_HCC__=1' if is_rocm_pytorch else ''],
-                                                 functions=['execute_aten_operator_address'],
-                                                 verbose=verbosity, with_cuda=True,
-                                                 build_directory=TORCH_CPP_BUILD_DIR)
+PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {{
+    m.def("execute_aten_operator_address", &execute_aten_operator_address, "Address of Aten operator executor");
+}}
+"""
 
-    C.register_aten_op_executor(str(aten_op_executor_cpp_extension.execute_aten_operator_address()))
+with open('aten_op_executor.cpp', 'w') as f:
```

Comment:
can we have the cpp part as a separate file? would be easier to format and review then

Reply:
sure!